### PR TITLE
test: New cli E2E test for columns flag

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira: [PSGO-498](https://keboola.atlassian.net/browse/PSGO-498)

**Changes:**
- added e2e test for `columns flag`

-----------

[PSGO-498]: https://keboola.atlassian.net/browse/PSGO-498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: test/cli/remote-create/table-columns-flag/args [hunk 1]
```diff
@@ -0,0 +1 @@
+remote create table --bucket in.c-bucket --name table1 --primary-key "" --columns 'col1,col2,col3' --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%%
```
Comment (michaljurecko) [inline test/cli/remote-create/table-columns-flag/args:1]: It should be `'"col1","col2","col3"'` instead of `'col1,col2,col3'`.

Comment (hosekpeter) [reply test/cli/remote-create/table-columns-flag/args]: I have tested many cases and this one stayed there

Comment (michaljurecko) [reply test/cli/remote-create/table-columns-flag/args]: Ok, but `'"col1","col2","col3"'` covers more problems, and it is use-case from the client ticket.

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**